scripts/make_confidence_report_bundled.py

```diff
@@ -0,0 +1,61 @@
+"""
+make_confidence_report_bundled.py
+Usage:
+  python make_confidence_report_bundled.py model.joblib
+
+  where model.joblib is a file created by cleverhans.serial.save containing
+  a picklable cleverhans.model.Model instance.
+
+This script will run the model on clean data and bundled adversarial examples
```

Comment:
I don't mind having scripts but could you explain how we should be deciding whether something is a tutorial or a script? Is a script a short example that we want to maintain?

Reply:
A script is a tool that we'll use frequently while actually doing work. A tutorial is something that is written for the purpose of showing how the library works, but is overly simplified and not practical for actual use (e.g. it builds a really small model).

Reply:
Sounds good, I'll make an issue to clarify this in the README file for future reference

Reply:
SGTM